packages/adblocker-benchmarks/Makefile

```diff
@@ -34,31 +35,31 @@ adblockpluscore: ./blockers/adblockpluscore/.git ./node_modules/adblockpluscore
 adblock-rs: ./node_modules/adblock-rs
 
 brave:
-	NODE_ENV=production node run.js brave requests.json
+	NODE_ENV=production node --expose-gc run.js brave $(FLAGS)
```

Comment:
Nit. It seems like `--expose-gc` and `--memory` could be only one flag since they have a similar role. Either `--memory` could have as a consequence to also inject `--expose-gc` or `--expose-gc` could be the flag and lead to memory measurements.

Reply:
`--expose-gc` is an option to `node` whereas `--memory` is an option to `run.js`. One way to do this is to parse the value of `FLAGS` in the makefile and then pass `--expose-gc` based on this. Another way is to require an explicit `NODE_FLAGS=--expose-gc` as an option to `make`. I don't know how to do the former at this point.

I've updated the patch to require `NODE_FLAGS=--expose-gc` now.

Let me know if you have any suggestions.

Reply:
Maybe something like this could be used in the Makefile:
```
NODE_FLAGS ?= --expose-gc
ifneq ($(FLAGS),--memory)
	undefine NODE_FLAGS
endif
```

Reply:
I thought of this, but then if you want to pass additional flags to `node` at some point then this would break.

If you're OK with this then I'll go ahead and do it.

On the other hand, someone using this code would likely use an alias anyway (`alias benchmark='make FLAGS=--memory NODE_FLAGS=--expose-gc'`).

Reply:
Another way would be to replace the generic `NODE_FLAGS` with specific flags such as `ENABLE_GC=1` in which case there is no ambiguity and you can easily control flags independently of each other.

Reply:
I see.

What about just a single `SHOW_MEMORY=1` in that case? If it's set, the makefile will pass the right options. In that case we can use `ifeq`.

Reply:
> What about just a single `SHOW_MEMORY=1` in that case? If it's set, the makefile will pass the right options. In that case we can use `ifeq`.

Sounds good!

Reply:
> What about just a single `SHOW_MEMORY=1` in that case?

Done.